openquake/commonlib/oqvalidation.py

```diff
@@ -56,7 +56,7 @@ class OqParam(valid.ParamSet):
     epsilon_sampling = valid.Param(valid.positiveint, 1000)
     export_dir = valid.Param(valid.utf8, None)
     export_multi_curves = valid.Param(valid.boolean, False)
-    exports = valid.Param(valid.Choice('csv', 'xml', 'geojson'), 'csv')
+    exports = valid.Param(str, 'csv')
```

Comment:
It would be good to do the validation here, instead of waiting for the export to fail